draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1372,59 +1325,36 @@ Note that a client may receive a response protected with a Security Context diff
 
       The client verifies the original countersignature SIGNATURE.
 
-   - If the verification of the countersignature fails, the client: i) SHALL stop processing the response; ii) SHALL NOT update the Response Number associated with the server, if the response is a non-notification response to a group request; and iii) SHALL NOT update the Notification Number associated with the server, if the response is an Observe notification {{RFC7641}}.
+   - If the verification of the countersignature fails, the client: i) SHALL stop processing the response; and ii) SHALL NOT update the Response Number associated with the server, if the response is received within a long exchange.
```

Comment:
The last sentence is perhaps correct, but gives the impression that response number can be updated if the response is not received within a long exchange. How about 
NEW
- If the verification of the countersignature fails, the client: i) SHALL stop processing the response; and ii) SHALL NOT update the Response Number associated with the server.

Reply:
I got the impression when reading through that there are other instances of redundant "in case of long exchange", I'll try to find them.

Reply:
Addressed in https://github.com/core-wg/oscore-groupcomm/pull/107/commits/f36e4cee74663796bae6e986400f8ae205e8ba1f